x-pack/plugins/ml/public/ui_actions/edit_swimlane_panel_action.tsx

```diff
@@ -39,20 +45,24 @@ export function createEditSwimlanePanelAction(
           '../embeddables/anomaly_swimlane/anomaly_swimlane_setup_flyout'
         );
 
-        const result = await resolveAnomalySwimlaneUserInput(
-          coreStart,
-          deps.data.dataViews,
-          embeddable.getInput()
-        );
-        embeddable.updateInput(result);
+        const { jobIds, viewBy, swimlaneType, panelTitle } = context.embeddable;
```

Comment:
When I add a new embeddable to a dashboard, the title I set in the config isn't getting applied to the panel:

<img width="240" alt="Screenshot 2024-03-21 at 13 02 48" src="https://github.com/elastic/kibana/assets/7405507/aac1bf3e-b8fb-4a6f-ba28-1c6444f0af69">

Reply:
Ah because it's still `title` for legacy embeddables and `panelTitle` with a new API. Fixed in https://github.com/elastic/kibana/pull/179073/commits/9ea94020d04b1bed650e1fa00cc8d35a05494fbe